src/components/notifications/AppNotifications/AppNotifications.scss

```diff
@@ -188,13 +185,13 @@
       color: var(--fg-color-1);
     }
     &__message {
-      color: var(--fg-color-2);
+      color: var(--fg-color-1);
       display: -webkit-box;
-      -webkit-line-clamp: 2;
+      -webkit-line-clamp: 3;
       -webkit-box-orient: vertical;
       overflow: hidden;
-      max-height: 2.75rem;
       margin-top: 0.25rem;
+      letter-spacing: -0.14px;
```

Comment:
Needed to override letter-spacing here. Once we have decided on the font family and new typography for the new designs. 

We could update many components that have custom font styling and just use `Text` component instead.